setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+import os.path
+import sys
+
+readme_path = os.path.join(os.path.dirname(__file__), 'README.md')
```

Comment:
No need to do all this. A simple `open('README.rst').read()` is enough. We don't have any back compatibility constraints. These constructs were used for older python versions which we have no reason to support.